com.zsmartsystems.zigbee/src/main/java/com/zsmartsystems/zigbee/zcl/protocol/ZclDataType.java

```diff
@@ -30,14 +30,7 @@
 import com.zsmartsystems.zigbee.zcl.field.WriteAttributeRecord;
 import com.zsmartsystems.zigbee.zcl.field.WriteAttributeStatusRecord;
 import com.zsmartsystems.zigbee.zdo.ZdoStatus;
-import com.zsmartsystems.zigbee.zdo.field.BindingTable;
-import com.zsmartsystems.zigbee.zdo.field.ComplexDescriptor;
-import com.zsmartsystems.zigbee.zdo.field.NeighborTable;
-import com.zsmartsystems.zigbee.zdo.field.NodeDescriptor;
-import com.zsmartsystems.zigbee.zdo.field.PowerDescriptor;
-import com.zsmartsystems.zigbee.zdo.field.RoutingTable;
-import com.zsmartsystems.zigbee.zdo.field.SimpleDescriptor;
-import com.zsmartsystems.zigbee.zdo.field.UserDescriptor;
+import com.zsmartsystems.zigbee.zdo.field.*;
```

Comment:
Is it possible to use the longer notation please - ie define each class. I thought that the code formatter should already do that and it avoids changing this code back and forward.